### DISCUSSION TITLE
Nemotron models

### DISCUSSION THREAD

**Discussion by BarfingLemurs** [ggml-org/llama.cpp#7954] [General]
Check out this model series: 

https://huggingface.co/nvidia/Nemotron-4-340B-Base

https://huggingface.co/nvidia/nemotron-3-8b-base-4k

The large set of models were released recently. They might run on a mac studio(s)

I'm guessing the performance is better than a previous very large model Falcon 180B.

I guess few people could run it, but it might be a good candidate for pruning and other tests, until a Llama 400B model shows up.

**Comment by daaain** [comment]
Is the new [Llama-3.1-Nemotron-51B](https://developer.nvidia.com/blog/advancing-the-accuracy-efficiency-frontier-with-llama-3-1-nemotron-51b/) the same family? It sounds really interesting, would be great if it was supported!
